static/js/place/place_template.tsx

```diff
@@ -749,7 +751,15 @@ class Chart extends Component<ChartPropType, ChartStateType> {
           <div id={this.props.id} className="svg-container"></div>
           <footer className="row explore-more-container">
             <div>
-              Data from <a href={config.url}>{config.source}</a>
+              <span>Data from </span>
+              {this.state.sources.map((source, index) => {
```

Comment:
we might need to add a filter to ensure uniqueness

Reply:
Checked the source in StatsData is a set so here should be good: https://github.com/datacommonsorg/website/blob/master/static/js/shared/data_fetcher.ts#L59